### PR TITLE
Update links to Kuberenetes api.yaml file to point to different repo

### PR DESCRIPTION
**Pull request by tkyi**
- [x] Moved Kubernetes example configs over to the `example-configs` repo, updating links in the guide to match
- [x] Removed dynamodb references
- [x] Update Kubernetes cluster setup doc

Blocked by https://github.com/screwdriver-cd-test/config-examples/pull/3

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by yahoocla** [comment]
CLA is valid!

File: docs/cluster-management/kubernetes.md [hunk 1]
```diff
@@ -111,13 +111,13 @@ Other environment variables can also be customized for Screwdriver. For a full l
 
 
 ## Deploy Screwdriver
-You can check out the `api.yaml` in the [Screwdriver Kubernetes repo](https://github.com/screwdriver-cd/kubernetes) for service and deployment definitions to run the Screwdriver API.
+You can check out the `api.yaml` in the [Screwdriver config examples repo](https://github.com/screwdriver-cd/config-examples) for service and deployment definitions to run the Screwdriver API.
```
Comment (d2lam) [inline docs/cluster-management/kubernetes.md]: This doesn't exist yet?

Comment (tkyi) [reply docs/cluster-management/kubernetes.md]: oops should be `screwdriver-cd-test`

File: docs/cluster-management/kubernetes.md [hunk 2]
```diff
@@ -111,13 +111,13 @@ Other environment variables can also be customized for Screwdriver. For a full l
...
 ## Deploy Screwdriver
-You can check out the `api.yaml` in the [Screwdriver Kubernetes repo](https://github.com/screwdriver-cd/kubernetes) for service and deployment definitions to run the Screwdriver API.
+You can check out the `api.yaml` in the [Screwdriver config examples repo](https://github.com/screwdriver-cd-test/config-examples) for service and deployment definitions to run the Screwdriver API.
 
 ### Create a Service
 A Kubernetes Service is an abstraction which defines a set of Pods and is assigned a unique IP address which persists.
 Follow instructions in [Creating a Service](http://kubernetes.io/docs/user-guide/connecting-applications/#creating-a-service) to set up your `service.yaml`.
 
-It should look like the Service in [api.yaml](https://github.com/screwdriver-cd/kubernetes/blob/master/api.yaml).
+It should look like the Service in [api.yaml](https://github.com/screwdriver-cd-test/config-examples/blob/master/kubernetes/api.yaml).
```
Comment (d2lam) [inline docs/cluster-management/kubernetes.md:120]: This link still doesn't work

**Review by d2lam — CHANGES_REQUESTED** [review]

**Review by minzcmu — CHANGES_REQUESTED** [review]
accident..

**Review by minzcmu — APPROVED** [review]

**Comment by d2lam** [comment]
Waiting on https://github.com/screwdriver-cd-test/config-examples/pull/3/files

**Review by d2lam — APPROVED** [review]

**Pull request merged**